src/prommis/uky/costing/tests/test_ree_costing.py

```diff
@@ -1206,9 +1279,14 @@ def test_costing_bounding(self, model):
         }
 
         for key in model.fs.costing.costing_lower_bound.keys():
-            assert model.fs.costing.costing_lower_bound[key].value == pytest.approx(
-                expected_costing_lower_bound[key], rel=1e-4
-            )
+            if model.fs.costing.costing_lower_bound[key].value <= 1e-4:
```

Comment:
Why the `if/else`? `pytest.approx` can take both the `rel` and `abs` argument at the same time.

```
model.fs.costing.costing_lower_bound[key].value == pytest.approx(
    expected_costing_lower_bound[key],
    abs=1e-8,
    rel=1e-4,
)
```